app/models/storage_location.rb

```diff
@@ -129,6 +129,10 @@ def self.import_inventory(filename, org, loc)
     adjustment.storage_location.increase_inventory(adjustment)
   end
 
+  def remove_empty_items
+    inventory_items.where(quantity: 0).delete_all
```

Comment:
Beware, delete_all skips callbacks. Is that ok here?

Reply:
Yep. Confirmed inventory items has no callbacks.